app/models/manageiq/providers/vmware/cloud_manager/orchestration_template.rb

```diff
@@ -97,6 +97,20 @@ def vm_param_groups(template)
             OrchestrationTemplate::OrchestrationParameterRange.new(:min_value => 4)
           ]
         ),
+        OrchestrationTemplate::OrchestrationParameter.new(
+          :name          => param_name('guest_customization', [vm_idx]),
+          :label         => 'Guest customization',
+          :description   => 'The computer name and network settings configured for this VM are applied to its Guest OS
+                             when the VM is powered on. The following settings are only applied the 1st time the VM is
```

Comment:
Can we go with this, please:

```
Check this to apply Hostname and NIC configuration for this VM to its Guest OS
when the VM is powered on. Guest OS must support this feature otherwise
provisioning will fail.
```

Other features that are mentioned in full text are not (yet) customizable via Service Dialog so we'd only confuse user telling her about them.